### DISCUSSION TITLE
Let's Remove Github Followers Spam Users

### DISCUSSION THREAD

**Discussion by devcrafta** [community/community#20208] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github-unfollowers.vercel.app/

**Comment by isinicolle** [comment]
It would be good

**Comment by radar-blip** [comment]
You can block other users who you dont want to follow you. You do this through settings on profile page.
Just click moderation under the access category. Add name of person you want to block.
Hope this helps!

**Comment by ghost** [comment, marked as answer]
Followers on GitHub are very important to many developers.
I think the link you provided is very useful and a good idea.
Thank you.
